Package.swift

```diff
@@ -397,6 +397,9 @@ let package = Package(
             dependencies: [
                 "BrowserServicesKit"
             ],
+            resources: [
+                .process("Resources")
```

Comment:
That’s interesting, according to Apple this shouldn’t be needed 🤔 

```
When you add a resource to your Swift package, Xcode detects common resource types for Apple platforms and treats them as a resource automatically. For example, you don’t need to make changes to your package manifest for the following resources:
Interface Builder files; for example, XIB files and storyboards
Core Data files; for example, xcdatamodeld files
Asset catalogs
.lproj folders you use to provide localized resources
If you add a resource file that Xcode doesn’t treat as a resource by default, you must configure it in your package manifest, as described in the next section.
```

Reply:
Yeah… apparently macOS does not like it without 🥲